zerver/models.py

```diff
@@ -214,8 +215,8 @@ def remote_user_to_email(remote_user):
 
 class RealmEmoji(models.Model):
     realm = models.ForeignKey(Realm)
-    name = models.TextField()
-    img_url = models.TextField()
+    name = models.TextField(validators=[MinLengthValidator(1), RegexValidator(regex=r'^[0-9a-zA-Z_\-\.]*$')])
+    img_url = models.URLField()
```

Comment:
Yes, that's the case here in particular and in most (if not all) cases when changing from `text` to `character varying` and vice versa. I think it would only be a problem if we already have URLs longer than 200 characters in existing databases. If you think this is worth testing or even extending the migration in order to handle such cases, let me know.

Maybe I should rather set a bigger `max_length` to the URLField in order to avoid such problems? A perfect solution would be to also check the existing URLs in the database against the URLValidator that Django provides.